Plugins/Public/base_plugin/Main.cpp

```diff
@@ -2530,6 +2560,93 @@ bool ExecuteCommandString_Callback(CCmds* cmd, const wstring &args)
 		//cmd->Print(L"OK Base is gone are you proud of yourself.");
 		return true;
 	}
+	else if (args.find(L"basedespawn") == 0)
```

Comment:
This was definitely in another PR

Reply:
Yep, admin commands, it'll dissapear from the diff once that one is merged.